src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+initial.orderStatusId=osId1
```

Comment:
I'd better set names of the initial statuses like `initial.orderStatus=completed` and so on.
The intention was to decouple the initial values and all available statuses.